test/unit/logic/round.js

```diff
@@ -752,6 +754,7 @@ describe('rounds', () => {
 				});
 			});
 		});
+		/* eslint-disable mocha/no-skipped-tests */
```

Comment:
Shouldn't this one be `eslint-enable` to reactivate it ?

Reply:
Good catch, thanks!